source/preferences/troubleshoot-notifications.rst

```diff
@@ -0,0 +1,144 @@
+----------------------------------
+
+Start by ensuring that your Mattermost preferences have notifications enabled:
+
+1. Select **Settings** |gear| in the top right corner of Mattermost.
+2. Select **Desktop and mobile notifications**.
+
+If **Send notifications for** is set to **Nothing**, then Mattermost notifications are currently disabled. Select either the **All new messages** or **Mentions, direct messages, and group messages** option instead.
+
+We recommend checking your Mattermost client settings next.
+
+Check your Mattermost client settings
+-------------------------------------
+
+.. tab:: Desktop
+
+    Ensure notifications are enabled in your Mattermost server connection.
+
+    1. Select the Mattermost Server option in the top left of the desktop app, then edit the server details.
+    2. Under **Permissions**, enable **Notifications**.
+
+    [add an animated GIF here]
+
+.. tab:: Mobile
+
+    Ensure that your Mattermost mobile preferences have notifications enabled.
+
+    1. Tap on your profile picture in the bottom right corner of Mattermost.
+    2. Tap **Settings** |gear|
+    3. Tap **Notifications**.
+    4. Tap **Push Notifications**.
+    5. Ensure that **All new messages** or **Mentions, direct messages, and group messages** option is selected.
+    6. Under **Trigger push notifications when…**, select **Online, away or offline** to always receive notifications.
+
+    [animated GIF here]
+
+    Also ensure that your mobile device isn't blocking device settings. Visit the **Android** or **iOS** tab below based on your mobile device.
+
+  .. tab:: Android
+
+    Ensure that your Android device isn't blocking Mattermost notifications by granting notification permission for Mattermost in device settings.
+
+    1. Open the Android **Settings app**, and tap **Application Manager**.
+    2. Locate **Google Play Services** and enable notifications for it. 
+    3. Locate **Mattermost** and enable notifications for it. 
+
+    Also ensure that your Android device isn't set to **Do Not Disturb** mode designed to block notifications.
+
+  .. tab:: iOS
+
+    Ensure that your iOS device isn't blocking Mattermost notifications by granting notification permission for Mattermost in device settings.
+
+    1. Open the iOS **Settings app** and tap **Notifications**.
+    2. In the list of apps, tap **Mattermost**. 
+    3. Enable the **Allow notifications** toggle and set **Notification Delivery** to **Immediate Delivery**.
+
+    Also ensure that your iOS device isn't set to **Do Not Disturb** mode or a **Focus mode** designed to block notifications.
+
+.. tab:: Web
+
+    If you prefer to use Mattermost in a web browser, you must grant notification permissions for Mattermost in the web browser. If you don't, the web browser can block you from receiving Mattermost notifications. Select the **Chrome**, **Edge**, **Firefox**, or **Safari** tab based on your preferred web browser:
+
+  .. tab:: Chrome
```

Comment:
Can we add these browser (Chrome, Edge...) tabs within the `<blockquote>` tag just above them? It makes their nested nature much clearer visually. I was trying it out in the inspect element and this is what it looked like -

<img width="765" alt="image" src="https://github.com/user-attachments/assets/1cde0596-770c-4c67-a60e-8dbf2862efe7">

Reply:
@cwarnermm We can add the below CSS anywhere in `mattermost-global.css` to this exact section and apply these changes. Tested that it works across devices and for both Light and Dark themes. 

```
#check-your-mattermost-client-settings .tab-set .tab-set {
    background-color: var(--color-background-secondary);
    padding: 1rem 1rem;
    margin-top: -2rem;
}
```

Can we do this as part of this PR?